app/payment/models/order.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+from django.db import models
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.content.models.event import Event
+from app.content.models.user import User
+from app.payment.enums import OrderStatus
+from app.util.models import BaseModel
+from app.util.utils import now
+
+
+class Order(BaseModel, BasePermissionModel):
+    access = AdminGroup.admin()
+    order_id = models.UUIDField(
+        auto_created=True, default=uuid.uuid4, primary_key=True, serialize=False
+    )
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="orders")
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name="orders")
```

Comment:
her er det nok kanskje smart med enten SET_NULL eller enda bedre PROTECT eller RESTRICT på on_delete, For vi vil gjene alltid beholde en historie på ordre

Reply:
Jeg tenkte det samme selv først. Men vil det være nødvendig å ha en historie om ordre for en bruker eller et event som ikke finnes lenger?

Og hvis jeg endrer til en PROTECT eller RESTRICT, vil det være mulig å slette et event uten å slette ordrene først da?

Men hvis du tenker det fortsatt er lurt å ha ordrestatistikken for slettede brukere og eventer så skal jeg endre.